eth/v1alpha1/node.proto

```diff
@@ -91,4 +98,18 @@ message Version {
 
 message ImplementedServices {
     repeated string services = 1;
-}
\ No newline at end of file
+}
+
+// Peers is a list of peer messages.
+message Peers {
+    repeated Peer peers = 1;
+}
+
+// Peer provides details of a peer on the network.
+message Peer {
+    // The address of the peer, as a full multiaddr, for example:
+    // /ip4/37.221.192.134/tcp/13000/p2p/16Uiu2HAm8maLMjag1TAUM52zPfmLbVMGFdwUAWgoHu1HDQLR6e17
+    string address = 1;
+    // The direction of the connection, must be either "inbound" or "outbound".
```

Comment:
should we use `enum` for direction status? 

like: https://github.com/prysmaticlabs/prysm/blob/master/proto/beacon/rpc/v1/services.proto#L165

Reply:
+1 enum would be better for direction. There can only be two types anyway

Reply:
Made this an enum.  Note I included an unknown status, because the first entry is used if the value is not available e.g. in the generated node.pb.go:

```
func (m *Peer) GetDirection() PeerDirection {
    if m != nil {
        return m.Direction
    }
    return PeerDirection_UNKNOWN
}
```